modules/oe_translation_poetry/config/install/oe_translation.oe_translation_request_type.poetry_translation_request.yml

```diff
@@ -0,0 +1,5 @@
+langcode: en
+status: true
+dependencies: {  }
+id: poetry_translation_request
```

Comment:
The entity type is called `oe_translation_request`. Why are we calling the bundle `poetry_translation_request`? Is `poetry` not enough?